modules/apps/headless/headless-admin-list-type/headless-admin-list-type-impl/src/main/java/com/liferay/headless/admin/list/type/internal/dto/v1_0/util/ListTypeEntryUtil.java

```diff
@@ -58,4 +61,14 @@ public static ListTypeEntry toListTypeEntry(
 		return listTypeEntry;
 	}
 
+	private static Map<Locale, String> _getLabelMap(
+		Map<Locale, String> localizedMap, String name) {
+
+		Locale siteDefaultLocale = LocaleUtil.getSiteDefault();
+
+		localizedMap.computeIfAbsent(siteDefaultLocale, key -> name);
+
+		return localizedMap;
+	}
```

Comment:
That change is not necessary.

Reply:
@marcelabc 

> Can you inline siteDefaultLocale?
> 
> Can we not have a new method? Since it is not complex, I don't think we need a method.

Can you check it?